### ISSUE TITLE
Vanilla install of Debian 10, FreeSWITCH mod_signalwire is not working

### ISSUE THREAD

**Issue by necival85** [signalwire/freeswitch#546]
Hello, I just installed a new server with Debian 10, and followed the Release Branch instructions at https://freeswitch.org/confluence/display/FREESWITCH/Debian+10+Buster

FreeSWITCH installed successfully:
```
root@localhost:~# fs_cli -x 'version'
FreeSWITCH Version 1.10.2-release-14-f7bdd3845a~64bit (-release-14-f7bdd3845a 64bit)
```

But I'm getting this error when I try to load mod_signalwire:

```
freeswitch@localhost> load mod_signalwire
+OK Reloading XML
-ERR [module load file routine returned an error]

2020-04-05 05:48:03.385731 [INFO] switch_stun.c:897 External ip address detected using STUN: 1.2.3.4
2020-04-05 05:48:03.465739 [INFO] switch_stun.c:897 External ip address detected using STUN: 1.2.3.4
2020-04-05 05:48:03.505722 [INFO] mod_enum.c:884 ENUM Reloaded
2020-04-05 05:48:03.505722 [INFO] switch_time.c:1433 Timezone reloaded 1750 definitions
2020-04-05 05:48:03.505722 [CRIT] switch_loadable_module.c:1786 Error Loading module /usr/lib/freeswitch/mod/mod_signalwire.so
**/lib/libsignalwire_client.so.1: undefined symbol: ks_rng_get_uint32**
```

I tried googling it but got no results. What dependency is missing?

Thank you

**Comment by andywolk** [comment]
What is your libks version?
`apt-cache policy libks`

**Comment by necival85** [comment]
I tried reinstalling the OS and installed FS from the "master"/unstable branch, but got the same error.


root@localhost:~# apt-cache policy libks
libks:
  Installed: 1.5.0-72~buster-72
  Candidate: 1.5.0-72~buster-72
  Version table:
 *** 1.5.0-72~buster-72 500
        500 http://files.freeswitch.org/repo/deb/debian-unstable buster/main amd64 Packages
        100 /var/lib/dpkg/status

**Comment by andywolk** [comment]
We have just released signalwire-client-c 1.3.0
This should fix the problem:
```
apt-get update
apt-get install signalwire-client-c
```

**Comment by necival85** [comment]
Works perfect now, thank you for the fast patch.
